pandas/tests/scalar/timedelta/test_timedelta.py

```diff
@@ -376,6 +379,20 @@ def test_unit_parser(self, units, np_unit, wrapper):
             result = Timedelta('2{}'.format(unit))
             assert result == expected
 
+    @pytest.mark.skipif(sys.version_info < (3, 5),
+                        reason="requires python3.5 or higher")
+    @pytest.mark.parametrize('unit', ['Y', 'y', 'M'])
+    def test_unit_m_y_deprecated(self, unit):
+        with tm.assert_produces_warning(FutureWarning) as w1:
+            Timedelta(10, unit)
+        msg = r'.* units are deprecated .*'
+        assert re.match(msg, str(w1[0].message))
+        with tm.assert_produces_warning(FutureWarning,
+                                        check_stacklevel=False) as w2:
+            to_timedelta(10, unit)
```

Comment:
Can you add a test for `to_timedelta` with multiple values as well? (eg `to_timedelta([1, 2], unit=unit)`

Reply:
Should this go in tests/indexes/timdeltas/test_tools.py - rather than add it here which is only for scalars ?

Reply:
Here or in the other file, either is fine for me